x-pack/test/cases_api_integration/security_and_spaces/tests/trial/cases/observables.ts

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import { OBSERVABLE_TYPES_BUILTIN } from '@kbn/cases-plugin/common/constants';
+import { getPostCaseRequest } from '../../../../common/lib/mock';
+import {
+  createCase,
+  deleteAllCaseItems,
+  addObservable,
+  similarCases,
+  updateObservable,
+  deleteObservable,
+  getCase,
+} from '../../../../common/lib/api';
+
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+
+  describe('observables', () => {
+    afterEach(async () => {
+      await deleteAllCaseItems(es);
+    });
+
+    describe('add observable to a case', () => {
+      it('allows the assignees field to be an empty array', async () => {
+        const postedCase = await createCase(supertest, getPostCaseRequest());
+        expect(postedCase.observables).to.eql([]);
+
+        const newObservableData = {
+          value: 'test',
+          typeKey: '0ad4bf8e-575f-49ad-87ea-8bcafd5173e4',
+          description: '',
+        };
+
+        const updatedCase = await addObservable({
+          supertest,
+          caseId: postedCase.id,
+          params: {
+            observable: newObservableData,
+          },
+        });
+
+        expect(updatedCase.observables.length).to.be.greaterThan(0);
+      });
+    });
+
+    describe('shows similar cases', () => {
```

Comment:
Could you please add a test where cases from different solutions (different `owner`) are not returned even if they have the same observables?